### DISCUSSION TITLE
Inconsistency in max char of prefix  - info icon message and validation logic

### DISCUSSION THREAD

**Discussion by arvind-dhariwal-zz** [Azure/data-management-zone#137] [General] [answered]
![image](https://user-images.githubusercontent.com/53206039/127886771-6a3c5dde-0a6f-4299-bb0c-a2ec6d6f9f88.png)

**Don't see this in Data management Zone** -

![image](https://user-images.githubusercontent.com/53206039/127887075-9b7c9310-34ab-46d6-a5ce-f61146ff219e.png)

**Comment by marvinbuss** [comment, marked as answer]
Thanks @arvind-dhariwal. This will be addressed in a PR in the next days. The issue can be tracked here: https://github.com/Azure/data-management-zone/issues/138

**Reply by marvinbuss** [reply]
Since the fix has been merged into `main`, I will mark this answer as solution.
